In [7]:
import os 
os.chdir("/Users/youheekil/Desktop/파이썬 포트폴리오 강의자료/프로젝트 3. Home Credit 대출상환여부 예측/Home_Credit_/input/")

In [8]:
import pandas as pd
df_train = pd.read_csv("train.csv")
df_test = pd.read_csv("test.csv")

In [11]:
df_train.columns

Index(['SK_ID_CURR', 'TARGET', 'CODE_GENDER', 'FLAG_OWN_CAR',
       'FLAG_OWN_REALTY', 'CNT_CHILDREN', 'AMT_INCOME_TOTAL', 'AMT_CREDIT',
       'AMT_ANNUITY', 'NAME_TYPE_SUITE', 'NAME_INCOME_TYPE',
       'NAME_EDUCATION_TYPE', 'NAME_HOUSING_TYPE',
       'REGION_POPULATION_RELATIVE', 'DAYS_BIRTH', 'DAYS_EMPLOYED',
       'DAYS_ID_PUBLISH', 'OWN_CAR_AGE', 'CNT_FAM_MEMBERS',
       'HOUR_APPR_PROCESS_START', 'ORGANIZATION_TYPE', 'EXT_SOURCE_1',
       'EXT_SOURCE_2', 'EXT_SOURCE_3', 'DAYS_LAST_PHONE_CHANGE',
       'AMT_REQ_CREDIT_BUREAU_YEAR'],
      dtype='object')

In [13]:
df_train.TARGET.value_counts()


0    18416
1     1584
Name: TARGET, dtype: int64

We see that there are 1584 instances with loan approaved. This is ~8.6% of the total number of samples from train dataset. Thus, we will keep the evaluation as AUC method since the dependant variable is skewed. Before we start modeling, for simplicity, we will be 